lib/main_common.pm

```diff
@@ -630,6 +630,16 @@ sub we_is_applicable {
       && get_var('MIGRATION_REMOVE_ADDONS', '') !~ /we/;
 }
 
+sub libreoffice_is_applicable {
+    # for opensuse libreoffice package has ExclusiveArch:  aarch64 %{ix86} x86_64
+    # do not know for SLE (so assume built for all)
```

Comment:
Seems SLE is only building it for x86_64 where desktop environment is present, so no problem here. Rest of the checks further will handle that